bin/deploy-pr.sh

```diff
@@ -31,6 +31,7 @@ if [ -n "$CI_PULL_REQUESTS" ] && [ "$WEBCHANGES" -ne 0 ]; then
   BUCKET_URI="s3://pr$PRNUM.hitech.eapd.cms.gov"
   BUCKET_POLICY='{"Version":"2012-10-17","Statement":[{"Sid":"PublicReadGetObject","Effect":"Allow","Principal":"*","Action":"s3:GetObject","Resource":"arn:aws:s3:::'"$BUCKET"'/*"}]}'
   aws s3 mb $BUCKET_URI
+  DEPLOYED_URL="http://$BUCKET.s3-website-us-east-1.amazonaws.com/"
```

Comment:
Pull this URL up and rename it from `URL` to something more meaningful

Reply:
Is this comment a TODO for yourself (I can't tell if you want to address this prior to merge or if we should open an issue or...)

Reply:
Correct, but also it wasn't a well-written message and lacked clear context.  😬